extensions/web-tracker/javascript/.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
I think we should ignore the snippet.min.js and the snippet.min.js.map instead of committing them WDYT ?

Reply:
yes, in this case, we can move snippet.min.js && snippet.min.js.map into dist, is it okay?

Reply:
I guess so, I'm not an expert at NPM packaging, would that make sense from that point of view?

Reply:
yes, I'll handle it, thanks